homeassistant/components/sensor/homematicip.py

```diff
@@ -0,0 +1,418 @@
+        """Return the name of the device."""
+        name = ''
+        if self._home.label != '':
+            name += self._home.label + ' '
+        name += self._device.label
+        if addon != '':
+            name += ' ' + addon
+        return name
+
+    @property
+    def name(self):
+        """Return the name of the generic device."""
+        return self._name()
+
+    @property
+    def icon(self):
+        """Return the icon of the generic device."""
+        return 'mdi:hexagon-outline'
+
+    @property
+    def state(self):
+        """Return the state of the generic device."""
+        return 'empty'
+
+    @property
+    def force_update(self):
+        """Force update."""
+        return True
+
+    @property
+    def should_poll(self):
+        """No polling needed."""
+        return True
+
+    @property
+    def available(self):
+        """Device available."""
+        return True
+
+    def _generic_state_attributes(self):
+        """Return the state attributes of the generic device."""
+        attr = {}
+        if hasattr(self._home.label, 'label'):
+            attr.update({ATTR_HOME: self._home.label})
+        if hasattr(self._device, 'label'):
+            attr.update({ATTR_LABEL: self._device.label})
+        if hasattr(self._device, 'lastStatusUpdate'):
+            last = self._device.lastStatusUpdate
+            if last is not None:
+                attr.update({ATTR_LASTSTATUS_UPDATE:
+                             last.isoformat()})
+        if hasattr(self._device, 'homeId'):
+            attr.update({ATTR_HOME_ID: self._device.homeId})
+        if hasattr(self._device, 'id'):
+            attr.update({ATTR_ID: self._device.id.lower()})
+        if hasattr(self._device, 'updateState'):
+            attr.update({ATTR_FIRMWARE: self._device.updateState.lower()})
+        if hasattr(self._device, 'firmwareVersion'):
+            attr.update({ATTR_ACTUAL_FIRMWARE: self._device.firmwareVersion})
+        if hasattr(self._device, 'availableFirmwareVersion'):
+            attr.update({ATTR_AVAILABLE_FIRMWARE:
+                         self._device.availableFirmwareVersion})
+        if hasattr(self._device, 'lowBat'):
+            attr.update({ATTR_LOW_BATTERY: self._device.lowBat})
+        if hasattr(self._device, 'unreach'):
+            attr.update({ATTR_UNREACHABLE: self._device.unreach})
+        if hasattr(self._device, 'sabotage'):
+            attr.update({ATTR_SABOTAGE: self._device.sabotage})
+        if hasattr(self._device, 'rssiDeviceValue'):
+            attr.update({ATTR_RSSI_DEVICE: self._device.rssiDeviceValue})
+        if hasattr(self._device, 'rssiPeerValue'):
+            attr.update({ATTR_RSSI_PEER: self._device.rssiPeerValue})
+        return attr
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes of the generic device."""
+        return self._generic_state_attributes()
+
+
+class HomematicipAccesspoint(HomematicipGenericDevice):
```

Comment:
This class now inherits the `force_update = True` which is wrong. This state should not have force updat.e